README.md

```diff
@@ -116,16 +114,20 @@ exemple of default template in JS :
 const { Kuzzle } = require('kuzzle-sdk');
 
 // instantiate a Kuzzle client
-const kuzzle = new Kuzzle('websocket', { host: 'kuzzle', autoReconnect: false });
+const kuzzle = new Kuzzle('websocket', {
+  host: 'kuzzle',
+  autoReconnect: false
+});
 
 // add a listener to detect any connection problems
 kuzzle.on('networkError', error => {
   console.error('Network Error:' + error);
-})
+});
 
-kuzzle.connect()
+kuzzle
+  .connect()
   .then(() => {
-    return [snippet-code]
+    return [snippet - code];
```

Comment:
This is a custom tag, I don't think it will still working with space around the `-`

Reply:
oh? Maybe that's why the test fail with a `SNIPPET NOT FOUND` err... :grimacing:

Reply:
:thinking: nope, it's just the README...